src/components/HeroImage/HeroImage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { s } from "./";
+
+export default function HeroImage(props){
+
+	const {
+		image: {
+			src,
+			mask: maskPath,
+			description
+		}
+	} = props;
+
+	// inline style to prevent the need for a huge png asset
+	const mask = { clipPath: `url(${maskPath})` };
+
+	return (
+		<div className={ s.hero } id="hero"> 	
+			<img 
+				className={ s.image }
+				src={ src } 
+				alt={ description }
+				style={ mask }
+			/>
+			<div className={ s.glow } id="glow"></div>
```

Comment:
```suggestion
			<div className={ s.glow } id="character__hero__glow"></div>
```
Again - `glow` is too generic a namespace, and we already know that we're going to have multiple to need to namespace them.